src/SDK/Trace/SpanProcessor/BatchSpanProcessor.php

```diff
@@ -100,10 +101,10 @@ public function shutdown(): bool
             return true;
         }
 
-        $this->running = false;
         if (null !== $this->exporter) {
             return $this->forceFlush() && $this->exporter->shutdown();
         }
+        $this->running = false;
```

Comment:
NB this is the important change for allowing flush on shutdown - setting `shutdown` too early meant that the forceFlush was a no-op